spec/dummy/config/initializers/address_lookup.rb

```diff
@@ -0,0 +1,8 @@
+EA::AddressLookup.configure do |config|
+  config.address_facade_server  = ENV["ADDRESS_FACADE_TEST_SERVER"]
+  config.address_facade_port    = ENV["ADDRESS_FACADE_TEST_PORT"]
```

Comment:
I'm a little confused. Is this initializer over-riding the settings used in `config/initializers/address_lookup.rb`